contentcuration/contentcuration/frontend/shared/vuex/channel/actions.js

```diff
@@ -2,6 +2,7 @@ import pickBy from 'lodash/pickBy';
 import { NOVALUE } from 'shared/constants';
 import { Channel, Invitation, ChannelUser } from 'shared/data/resources';
 import client from 'shared/client';
+import { debouncedSyncChanges } from 'shared/data/serverSync';
```

Comment:
How about these inline linting reports huh?!

Reply:
Oh yeah, my linter isn't catching anything on the pre-commit hook